apps/meteor/ee/app/livechat-enterprise/server/api/contacts.ts

```diff
@@ -9,17 +11,15 @@ const ajv = new Ajv({
 });
 
 type blockContactProps = {
-	visitorId: string;
+	visitor: ILivechatContactVisitorAssociation;
 };
 
 const blockContactSchema = {
 	type: 'object',
 	properties: {
-		visitorId: {
-			type: 'string',
-		},
+		visitor: ContactVisitorAssociationSchema,
 	},
-	required: ['visitorId'],
+	required: ['visitorId', 'source'],
```

Comment:
```suggestion
	required: ['visitor'],
```

Reply:
no, this is right; you can't block a channel without specifying both visitor and source.

Reply:
OHH I got the issue now.